zipkin-lens/src/features/dependencies/Edge.ts

```diff
@@ -11,8 +11,14 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-import createReducer from '../reducers';
 
-type RootState = ReturnType<ReturnType<typeof createReducer>>;
+type Edge = {
```

Comment:
Edge was defined in `NodeDetailData.tsx`, but it is now in a separate file Transferred.